packages/manager/modules/nutanix/src/dashboard/general-info/redeploy/template.html

```diff
@@ -45,12 +45,12 @@ <h2 data-translate="nutanix_redeploy_title"></h2>
             <ul class="list-unstyled">
                 <li>
                     <span
-                        data-ng-bind=":: $ctrl.CLUSTER_CONFIG_TERMS.PRISM_CENTRAL + ' :'"
+                        data-translate="{{('nutanix_redeploy_prism_central' | translate) + ' :'}}"
```

Comment:
i think you can include the `:` with translation.
```suggestion
                        data-translate="nutanix_redeploy_prism_central"
```